src/main/java/Duke.java

```diff
@@ -1,10 +1,121 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    public enum commands {
+        BYE, LIST, MARK, UNMARK, TODO, DEADLINE, EVENT, DELETE;
+
+        @Override
+        public String toString() {
+            return name().toLowerCase();
+        }
+    }
+
+    private static String numOfTasks(ArrayList<Task> tasks) {
+        return tasks.size() == 1 ? " task " : " tasks ";
+    }
+
     public static void main(String[] args) {
+        ArrayList<Task> tasks = new ArrayList<>();
+        Scanner sc = new Scanner(System.in );
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello I'm Duke!\n" + "How can I help you?");
+
+        while (true) {
+            try {
+                String input = sc.nextLine();
+                String[] splitInput = input.split(" ", 2);
+                String command = splitInput[0];
+                String remainingInput = splitInput.length > 1 ? splitInput[1] : null;
+
+                if (command.equals(commands.BYE.toString())) {
+                    System.out.println("Bye. Hope to see you again!");
+                    break;
+                } else if (command.equals(commands.LIST.toString())) {
+                    System.out.println("Here are the tasks in your list");
+                    if (tasks.size() != 0) {
+                        for (int i = 0; i < tasks.size(); i++) {
+                            System.out.println((i + 1) + "." + tasks.get(i).toString());
+                        }
+                    }
```

Comment:
Perhaps the if condition can be taken out so that it is easier to see the "happy path" and reduce indentation of code.
```suggestion
                    if (tasks.size() == 0) {
                        continue;
                    }
                    for (int i = 0; i < tasks.size(); i++) {
                        System.out.println((i + 1) + "." + tasks.get(i).toString());
                    }
```